python/lsst/ts/ATHexapod/controller.py

```diff
@@ -97,13 +121,13 @@ def is_connected(self):
             Command to send to hexapod. A 'newline' character will be appended.
         has_response : `bool`
             Does the command have a response?
-        multi_line : `bool`
-            Is the response received in multiple lines?
+        num_line : `int`
+            The number of expected lines/replies.
```

Comment:
Consider combining has_response into num_line by using 0 for no responses.

This makes the API easier to understand in that the user doesn't have to wonder what happens if they specify has_response=False and num_line > 0.